db/data/regions/_default/en.yml

```diff
@@ -91,75 +100,96 @@ en:
             optional: Postal code (optional)
           errors:
-            blank: Enter a postal code
+            blank_instructional: Enter a postal code
+            blank_informative: Postal code is missing
             blocked_address: This location isn't supported.
-            invalid: Enter a valid postal code
-            invalid_for_country: Enter a valid postal code for %{country}
-            invalid_for_province: Enter a valid postal code for %{province}
-            invalid_for_country_and_province: Enter a valid postal code for %{province},
-              %{country}
+            invalid_instructional: Enter a valid postal code
+            invalid_informative: Postal code isn't valid
+            invalid_for_country_instructional: Enter a valid postal code for %{country}
+            invalid_for_country_informative: Postal code doesn't match %{country}
+            invalid_for_province_instructional: Enter a valid postal code for %{province}
+            invalid_for_province_informative: Postal code doesn't match %{province}
+            invalid_for_country_and_province_instructional: Enter a valid postal code for %{province}, %{country}
+            invalid_for_country_and_province_informative: Postal code doesn't match %{province}, %{country}
             too_long: The postal code value is too long
             contains_emojis: Postal code cannot contain emojis
             contains_mathematical_symbols: Postal code cannot contain mathematical symbols
             contains_restricted_characters: Postal code can only contain letters,
               numbers, local characters, and special characters
-            not_supported: We don't currently support this country/region. Enter a
-              new address and try again.
-            contains_html_tags: Postal code cannot contain HTML tags.
-            unknown_for_province: Enter a valid postal code for %{province}
-            unknown_for_address: Postal code may be incorrect.
+            not_supported_instructional: Postal code not supported. Enter a new address and try again
+            not_supported_informativel: Postal code not supported
+            contains_html_tags: Postal code can't contain HTML
+            unknown_for_province_instructional: Enter a valid postal code for %{province}
+            unknown_for_province_informative: Postal code is invalid for %{province}
+            unknown_for_address: Postal code may be incorrect
         phone:
           label:
             default: Phone
             optional: Phone (optional)
           errors:
-            blank: Enter a phone number
-            invalid: Enter a valid phone number
+            blank_instructional: Enter a phone number
+            blank_informative: Phone number is missing
+            invalid_instructional: Enter a valid phone number
+            invalid_informative: Phone number code isn't valid
             too_long: Phone is too long (maximum is 255 characters)
```

Comment:
Hmm I think a phone number would be too long if it were 235 characters shorter than this, seems kind of useless as is, no?

Reply:
This is changing as it is now.. but I agree.. this message is wrong.